### PR TITLE
Solved #4: solve exercises of Making Choices

### PR DESCRIPTION
**Pull request by Jainil2004**
*Issue: #4 - solve exercises of Making Choices*
**Issue resolved**
**commit 1**: adds the detailed solutions
**commit 2**: makes the overall code solutions more readable and understandable

### CODE REVIEW

File: Making Choices/exercise-solutions.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    print(ph ,"It's acidic!")
+elif ph < 3.0:
+    print(ph, "is VERY acidic ! Be careful.")
+# this prints the wrong message when a ph of 2 is entered. what is the 
+# problem and how can you fix it?
+
+# solution: the problem is that if the first statement is true then the second
+# statement will not be checked. as elif works in a way that if the first
+# statement is false only then check the elif statement. so we can fix it by
+# changing the elif statement to if statement. corrected code:
```
Comment (karsterr) [inline Making Choices/exercise-solutions.py]: Code is true yes, but there is one thing. Both messages displayed when code is executed. Question isn't specifically wanted single output but I think single output is better than two outputs that looks same. Here is my recommendation:

```python
if ph < 7.0:
    if ph < 3.0: # This code block will be executed when ph < 3
        print(ph, "is VERY acidic ! Be careful.")
    else:
        print(ph ,"It's acidic!") # This line will be executed when ph > 3
```
First if checks ph if it's lower than 7.0. If it's true the next if checks if ph is lower than 3.0. If it's true, program gives outputs text about very acidic. If its not the case, normal acidic text will be printed.

File: Making Choices/exercise-solutions.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+# the following code displays a messages about the acidity of a solution:
+ph = float(input("Enter the pH level: "))
+if ph < 7.0:
+    print("It's acidic!")
+elif ph < 4.0: 
+    print("its a strong acid!")
+
+# a. what message are displayed when the user enters 6.4?
+# solution: the output will be "It's acidic!"
+# b. what message are displayed when the user enters 3.6?
```
Comment (karsterr) [inline Making Choices/exercise-solutions.py:224]: Solution of b is missing

**Review by karsterr — CHANGES_REQUESTED** [review]

File: Making Choices/exercise-solutions.py (hunk 1 above)
Comment (Jainil2004) [reply Making Choices/exercise-solutions.py]: issue noted, working on the fix

File: Making Choices/exercise-solutions.py (hunk 2 above)
Comment (Jainil2004) [reply Making Choices/exercise-solutions.py:225]: issue noted, working on the fix

**Comment by Jainil2004** [comment]
issue #4 - solve exercises of Making choices 
minor Issues found - 
1. incorrect output in Problem 9
2. solution B of problem 10 missing 

issues resolved? 
YES

Changes Made: 
- fixed the corrected code output in problem 9 : it no longer displays both the messages when both conditions are true
- added solution of B part of problem 10

**Pull request merged**